db/models.py

```
@@ -4,7 +4,7 @@
 import pymongo
 import pytz
 from beanie import Document, Link
-from pydantic import BaseModel, Field, ConfigDict, field_validator
+from pydantic import BaseModel, Field, ConfigDict, field_validator, model_validator
```

Ruff

`pydantic.model_validator` imported but unused

Remove unused import: `pydantic.model_validator`

(F401)